sys/include/ztimer.h

```diff
@@ -197,6 +197,11 @@
  *              if if these are missing it will use same basic timer
  *              as ZTIMER_USEC does.
  *
+ * If `periph_rtt` or is required with direct access by another MODULE or
+ * application, `ztimer_periph_rtt_auto_select` can be disabled to avoid automatic
+ * selection of `ztimer_periph_rtt` as a backend for ZTIMER_SEC and ZTIMER_MSEC.
+ * i.e.: `DISABLE_MODULE += ztimer_periph_rtt_auto_select`.
```

Comment:
Could we maybe invert the logic? Making a module that avoids the default selection if present in the application makefile? then we wouldn't have the issue of the default module always showing up, right?

Reply:
That was the initial approach I had, I can go back to that.

Reply:
@kfessel thoughts?

Reply:
i think the was a problem that lead to abondoning that approach (like could not be kconfig model???)

Reply:
I split into an alternative the old versio so its easier to compare https://github.com/RIOT-OS/RIOT/pull/17284

Reply:
@leandrolanzieri @kfessel opinion on what option to prefer this one or https://github.com/RIOT-OS/RIOT/pull/17284?